std/traits.d

```diff
@@ -7664,12 +7664,20 @@ template getSymbolsByUDA(alias symbol, alias attribute) {
                   .format(names[0]));
     }
 
+    // filtering inaccessible members
+    enum noInaccessibleMembers(string name) = (__traits(compiles, __traits(getMember, symbol, name)));
```

Comment:
double negation is wired.
`isAccessible` 
also the `()` around the `__traits`-Expression are not needed.